<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonthCal</class>
 <widget class="QMainWindow" name="MonthCal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="yearlabel">
          <property name="text">
           <string>Год</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="year"/>
        </item>
        <item>
         <widget class="QLabel" name="monthlabel">
          <property name="text">
           <string>Месяц</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="month"/>
        </item>
        <item>
         <widget class="QPushButton" name="choose">
          <property name="text">
           <string>Выбрать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Выделить особые даты</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="monthName">
        <property name="text">
         <string>Ниже будет выведен календарь</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="back">
        <property name="text">
         <string>Ввернуться на главный экран</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
